### PR TITLE
Classifier display

### PR DESCRIPTION
**Pull request by jzhang-gp**
Display threshold by default on the task page. All thresholds can be updated on the page. 

image

### CODE REVIEW

File: backend/models.py [hunk 1]
```diff
@@ -59,33 +59,32 @@ def export_new_raw_data():
 
 @bp.route('update_model_deployment_config', methods=['POST'])
 def update_model_deployment_config():
+    logging.error(request.form)
```
Comment (edu-gp) [inline backend/models.py:62]: Do we need this?

File: backend/templates/tasks/show.html [hunk 2]
```diff
@@ -608,34 +620,36 @@ <h4>Deployment Configurations</h4>
...
+      div_radio.style.display = 'none';
+  }
 
   function displayRadioButton(id) {
     let checkbox = document.getElementById(id);
     let model_id = id.split("_")[0];
     let radio_button = document.getElementById(model_id + "_radio");
     let div_radio = document.getElementById(model_id + "_div_radio");
-    let div_threshold = document.getElementById(model_id + "_div_threshold");
+    {#let div_threshold = document.getElementById(model_id + "_div_threshold");#}
```
Comment (edu-gp) [inline backend/templates/tasks/show.html:647]: Can we delete this comment?

File: backend/templates/tasks/show.html [hunk 3]
```diff
@@ -608,34 +620,36 @@ <h4>Deployment Configurations</h4>
...
     if (checkbox.checked) {
       div_radio.style.display = 'block';
     } else {
       div_radio.style.display = 'none';
-      div_threshold.style.display = 'none';
+      {#div_threshold.style.display = 'none';#}
```
Comment (edu-gp) [inline backend/templates/tasks/show.html:652]: ditto

File: backend/templates/tasks/show.html [hunk 4]
```diff
@@ -539,56 +539,68 @@ <h4>Deployment Configurations</h4>
...
+                                <label for="selected_model_id">Selected for deployment</label>
+                              </div>
+{#                              <div id="{{mv.id}}_div_threshold" style="display: none">#}
+{#                                <label for="threshold">Threshold</label>#}
+{#                                {% if deployment_config %}#}
+{#                                    <input type="text" name="threshold" value="{{ deployment_config["threshold"] }}" />#}
+{#                                {% else %}#}
+{#                                    <input type="text" name="threshold" value="0.5" />#}
+{#                                {% endif %}#}
+{#                              </div>#}
```
Comment (edu-gp) [inline backend/templates/tasks/show.html:599]: There are a lot of commented out code here, can we delete them?

File: backend/models.py (hunk 1 above)
Comment (jzhang-gp) [reply backend/models.py:62]: Removed.

File: backend/templates/tasks/show.html (hunk 2 above)
Comment (jzhang-gp) [reply backend/templates/tasks/show.html:647]: Done.

File: backend/templates/tasks/show.html (hunk 3 above)
Comment (jzhang-gp) [reply backend/templates/tasks/show.html:652]: Done.

File: backend/templates/tasks/show.html (hunk 4 above)
Comment (jzhang-gp) [reply backend/templates/tasks/show.html:599]: Done.

**Pull request merged**